### PR TITLE
add back David's name into the authors list

### PR DESCRIPTION
**Pull request by rahulanand16nov**
I think @davidor 's name was removed mistakenly. Though he's not going to contribute still he's still an author, right? :)

### CODE REVIEW

File: limitador-server/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "limitador-server"
 version = "0.5.1"
-authors = ["Eguzki Astiz Lezaun <eguzki@redhat.com>"]
+authors = ["Eguzki Astiz Lezaun <eguzki@redhat.com>", "David Ortiz <z.david.ortiz@gmail.com>"]
```
Comment (eguzki) [inline limitador-server/Cargo.toml:4]:
```suggestion
authors = ["Rahul Anand <rahanand@redhat.com>", "Eguzki Astiz Lezaun <eguzki@redhat.com>", "David Ortiz <z.david.ortiz@gmail.com>"]
```

**Comment by rahulanand16nov** [comment]
@davidor I think current maintainers should be first on the list, is it okay with you? Can we get ownership of https://crates.io/crates/limitador so we can push the new release?

**Comment by rahulanand16nov** [comment]
Let's wait for @davidor's reply before merging this.

**Comment by eguzki** [comment]
@alexsnaps should be also added

**Comment by eguzki** [comment]
@rahulanand16nov you did not accept my suggested change

**Comment by rahulanand16nov** [comment]
@eguzki I haven't contributed much yet but will add once I have contributed in a bit more significant way :sweat_smile:

**Pull request merged**